website_sale_vat_required/static/src/js/website_sale_vat_required.tour.js

```diff
@@ -9,22 +9,38 @@ odoo.define("website_sale_vat_required.tour", function (require) {
 
     steps.splice(
         _.findIndex(steps, {content: "go to checkout"}) + 1,
-        0,
+        4,
```

Comment:
This makes no sense to me and could cause issues with other modules. You are taking this chunk of the orignial test away: https://github.com/odoo/odoo/blob/a0c1e2aa602ae46598a350ea6ae8d8b4a0c1c823/addons/website_sale/static/tests/tours/website_sale_buy.js#L63-L87

I'm thinking that `website_sale_checkout_skip_payment` should `splice` its part so we don't have to be aware of it on each tour that uses this

Reply:
This test already has its own steps :)

Reply:
Rerun Odoo tests, maybe there was a temporary glitch making them fail on a tour

Reply:
It's always the same...